tools/data-handler/test/utils/clingo-parser.test.ts

```diff
@@ -76,6 +76,13 @@ describe('ClingoParser', () => {
     expect(result.results[0].fieldName).to.equal(fieldValue);
   });
 
+  it('should parser field correctly when last argument is an empty string', async () => {
```

Comment:
The test description does not make sense.
Maybe,  'should parser **handle** field correctly when last argument is an empty string'?